packages/front-generator/src/generators/react-typescript/entity-management/template/EntityManagementEditor.tsx.ejs

```diff
@@ -109,6 +109,10 @@ class <%= editComponentName %>Component extends React.Component<Props & WrappedC
       return <Spinner/>;
     }
 
+    if (status === "ERROR") {
```

Comment:
Just thinking aloud: maybe we could rename it to `LOAD_ERROR` or something similar? So that it's more implicit what kind of error are we dealing with?

Reply:
Agree. We are using this statuses both on entity and list components. Maybe add a set of error types. @web-devel  any thoughts?